crypto/asn1/a_mbstr.c

```diff
@@ -342,54 +341,8 @@ static int cpy_utf8(unsigned long value, void *arg)
     return 1;
 }
 
-/* Return 1 if the character is permitted in a PrintableString */
-static int is_printable(unsigned long value)
-{
-    int ch;
-    if (value > 0x7f)
-        return 0;
```

Comment:
Is this significant comparison? In sense that is it possible that doing something else might mask some specific condition that shouldn't be masked? Note that input to typestr can actually be wider than char...